test/com/ka/spreadsheet/diff/TestUtils.java

```diff
@@ -51,7 +51,7 @@ public static void assertEquals(Object actual, Object expected) {
   public static void assertEquals(String messagePrefix, Object actual, Object expected) {
     if (((actual == null && expected != null) || (actual != null && expected == null))
         || !actual.equals(expected)) {
-      throw new AssertionError(messagePrefix + "\nactual: " + actual + "\nexpected: " + expected);
+      throw new AssertionError(messagePrefix + "\nactual  : " + actual + "\nexpected: " + expected);
```

Comment:
this change intentional?

Reply:
Yes - it makes the actual and expected text line up vertically when the assertion message is displayed (because `"actual  ".length()` == `"expected".length()`) .  I had a coding error that caused one of these to be thrown, and it was a subtle text difference, and I couldn't see it.  Aligning them made the difference.  Accept or reject as you like,

Reply:
Sounds good!